src/layouts/default.pug

```diff
@@ -1,3 +1,4 @@
+- var iconCount = 0
```

Comment:
That's a bit dirty here.
Is there a cleaner way to do that?

Like moving this to the `icons/layout.pug` ?
Or adding it to the `helpers.` for jade?